vmdb/app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -34,12 +42,12 @@ def textual_ipaddress
     {:label => "IP Address", :value => @ems.ipaddress}
   end
 
-  def textual_type
-    {:label => "Type", :value => @ems.emstype_description}
+  def textual_port
+    @ems.supports_port? ? {:label => "Port", :value => @ems.port} : nil
   end
 
-  def textual_port
-    @ems.supports_port? ? {:label => "API Port", :value => @ems.port} : nil
+  def textual_zone_id
```

Comment:
@erinboyd can you rename this method to textual_zone, i think that will make more sense.